doc/sphinx-guides/source/api/changelog.rst

```diff
@@ -5,25 +5,26 @@ API Changelog
     :local:
     :depth: 1
 
-v6.1
-----
+6.2
+---
+Changes
+~~~~~~~
+- **/api/datasets/{id}/versions/{versionId}**: The includeFiles parameter has been renamed to excludeFiles. The default behavior remains the same, which is to include files. However, when excludeFiles is set to true, the files will be excluded. A bug that caused the API to only return a deaccessioned dataset if the user had edit privileges has been fixed.
+- **/api/datasets/{id}/versions**: The includeFiles parameter has been renamed to excludeFiles. The default behavior remains the same, which is to include files. However, when excludeFiles is set to true, the files will be excluded.
 
-New
-~~~
-- **/api/dataverses/{id}/datasetSchema**: See :ref:`get-dataset-json-schema`.
-- **/api/dataverses/{id}/validateDatasetJson**: See :ref:`validate-dataset-json`.
+6.1
+---
 
 New
 ~~~
 - **/api/admin/clearThumbnailFailureFlag**: See :ref:`thumbnail_reset`.
```

Comment:
Why is there any "New" at all in this file? There shouldn't be. We took it out: https://guides.dataverse.org/en/6.1/api/changelog.html

Does "develop" need to be merged into this branch?

Reply:
The file was just updated with the latest version and changes 😄